### ISSUE TITLE
Error after installing version 2.2.0-beta1 of PSReadLine, PowerShell crashed

### ISSUE THREAD

**Issue by william-monroy** [PowerShell/PSReadLine#3038]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
### Exception

System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```
```


### Screenshot

![terminal_bug](https://user-images.githubusercontent.com/58092741/141826625-beb1606a-4f64-45ed-8bb7-5fa9f2c98cc1.png)


### Environment data

```console
### Environment
PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.22000
BufferWidth: 120
BufferHeight: 30

Last 0 Keys


```
```


### Steps to reproduce

I followed the steps in this blog: [https://www.hanselman.com/blog/my-ultimate-powershell-prompt-with-oh-my-posh-and-the-windows-terminal](url) and everything was fine until I got an error when I tried to execute this line in my script: **"Set-PSReadLineOption -PredictionViewStyle ListView"**. The sequence of steps that happened to get this error was to follow step by step the following tutorial: [https://www.youtube.com/watch?v=VT2L1SXFq9U](url) from minute 37:37 and my error occurred when I tried to do the step at minute 41:56

### Expected behavior

Complete and error-free installation of **PSReadLine 2.2.0-beta1** and use this functionality in my PowerShell:

![image](https://user-images.githubusercontent.com/58092741/141828519-63d075f7-1db3-4145-821c-6c95a96fa0b6.png)


### Actual behavior

My PowerShell crashed, it is no longer possible to use it.

![image](https://user-images.githubusercontent.com/58092741/141828777-1dc518c1-31b7-4eed-a7dc-ad5a360a5dd8.png)

**Comment by andyleejordan** [comment]
Hi, I see you're using an old version of PSReadLine. Please upgrade to use the latest [2.2.0-beta4 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta4) and let us know if this continues to reproduce. Instructions for doing so:

1. stop all instances of `pwsh`.
2. from `cmd.exe` run: `pwsh -noprofile -command "Install-Module PSReadLine -AllowPrerelease -Force"`

--------

If you want to remove that beta version of PSReadLine and use the 2.1.0 version of PSReadLine that's shipped with PowerShell 7.2, you can:

1. run `pwsh -noprofile -noninteractive` to start `pwsh` without loading PSReadLine
2. run `Uninstall-Module -Name PSReadLine -RequiredVersion <2.2.0-beta1 or 2.2.0-beta2> -AllowPrerelease` to remove the module. Or, you can manually remove that module folder.

Thanks!

**Comment by william-monroy** [comment]
Now works perfect! Thank you so much @andschwa 

![image](https://user-images.githubusercontent.com/58092741/141829834-e3d2c1d7-5b14-414f-8f99-ecc3d6216b3f.png)
